src/exchange/exchange_client.rs

```diff
@@ -79,6 +80,7 @@ impl ExchangeClient {
         wallet: LocalWallet,
         base_url: Option<BaseUrl>,
         meta: Option<Meta>,
+        spot_meta: Option<SpotMeta>,
```

Comment:
I don't think there's any need to support passing in a meta and I'd rather not make a breaking api change

Reply:
Changed this to not pass in spot_meta